storage/database/dynamodb.go

```diff
@@ -56,6 +57,7 @@ var noTableNameErr = errors.New("dynamoDB table name not provided")
 const dynamoWriteSizeLimit = 399 * 1024 // The maximum write size is 400KB including attribute names and values
 const dynamoBatchSize = 25
 const dynamoMaxRetry = 5
```

Comment:
`Increase maxRetries from 3 (default) to 5`
There is no change, right? @aidan-kwon

Reply:
`dynamoMaxRetry` was used only for batch write. Other dynamoDB client operations had a default maxRetry value, 3. With this change in line 163, `dynamoMaxRetry` is applied for all dynamoDB client operations.

Reply:
```suggestion
const dynamoMaxRetry = 5 // used in DynamoDB client and batchwrite
```

Reply:
Does it mean everywhere using dynamoDB?